package.json

```diff
@@ -55,7 +55,7 @@
     "ember-cli-babel": "^7.21.0",
     "ember-cli-htmlbars": "^5.2.0",
     "ember-cli-typescript": "^3.1.4",
-    "ember-concurrency": "^1.2.1",
+    "ember-concurrency": "^1.0.0 <3",
```

Comment:
according to the [Online Semver checker](https://jubianchi.github.io/semver-check/#/^1.0.0%20<3/2.0.0) this does not quite work, because `2.0.0` is not matching that constraint

Reply:
oh whoops, yep. that should be `>=1.0.0 <3`

Reply:
I believe you can do `"^1.0.0 | ^2.0.0"`

Reply:
@cibernox I just updated it to use `>=1.0.0 <3` but let me know if you have a preference for `"^1.0.0 || ^2.0.0-beta.1"`

Reply:
Not really, I believe both work.